boxoffice/views/participant.py

```diff
@@ -77,9 +91,16 @@ def assign(order):
                 details=assignee_details,
                 line_item=line_item,
             )
+
             db.session.add(new_assignee)
             db.session.commit()
             send_ticket_assignment_mail.queue(line_item.id)
+
+            if old_assignee is not None:
+                # Send notification to previous assignee
+                send_ticket_reassignment_mail.queue(
+                    line_item.id, old_assignee.id, new_assignee.id
+                )
         return {'status': 'ok', 'result': {'message': 'Ticket assigned'}}
```

Comment:
Also requires `_(...)`

Reply:
Unless the front-end is treating 'Ticket assigned' as a literal string. @vidya-ram?

Reply:
This string is not used in the front-end